### PR TITLE
#239/roadmap

### LINKED ISSUE #239: Publish Product Roadmap 🗺️ 📈
# Expected Behaviour

In order to allow the contributing community to focus submissions towards a faster development of planned releases we need to provide guidance in our documentation on what are the expected feature builds for oncoming versions of Kindly.

# Current Behaviour

The only guidance on what work is expected comes from Issues already open. New collaborators are likely to avoid opening new issues as they won't be clear on what direction the team is hoping to move the product.

## ~~Steps to Reproduce~~ Suggestion
- We should publish a static grab of the product roadmap along with expected user stories
- New issues should be opened in line with the Milestones set in the Roadmap

### PR DESCRIPTION
**Pull request by camelPhonso**
# Description

**Closes #239**  

This PR publishes information on the new Product Roadmap and connected User Stories to help guide contributions towards the next planned releases.

### Files changed

- `CONTRIBUTING.md` - added an image of the Roadmap and a new Header with the new User Stories broken down by Milestone
- `roadmap.png` - screengrab of the roadmap for documentation

### UI changes

No changes to the UI 🙅 

### Changes to Documentation

This is a documentation PR - all changes are outlined above

# Tests

No changes to the behaviour of the code, so no tests were needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,9 @@ Kindly is an open source web application created by Trafalgar Girls and enBloc,
 
 <h2 align=center>Contributing</h2>
 
-We **welcome** your contributions. Please familiarise yourself with the following documentation before getting started. Contributing docs can be found in [kindly/.github](https://github.com/enBloc-org/kindly/tree/dev/.github)
-
-You can also connect with the developing community on our [discord server](https://discord.gg/Mrh5NYw52S)
+We **welcome** your contributions. Please familiarize yourself with our documentation before getting started. You can find guidance on how to contributing in the following sources:
```
Comment (eliazzo) [inline README.md:10]: You can find guidance on how to contributing* in the following sources:

*contribute

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+- As a User I want to be able to suspend my account
+- As a User I want to be able to delete my account
+- As a User I want to be able to browse through Kindly before I need to create an account in order to interact Donations
+- As a User I want to be able to sign up and login to Kindly with my login from social platforms such as Google or Facebook
+- As a User I want to be able to keep my account logged in between visits to Kindly 
+
+### Donations Milestone 🤝
+- As a Donor I want to be able to mark any of my items as 'Reserved'
+- As a Donor I want to prevent new Inquiries about any Items I have marked as 'Reserved'
+- As a Refugee I want to receive confirmation what an Item has been reserved for me upon request
```
Comment (eliazzo) [inline .github/CONTRIBUTING.md:28]: This is potentially not our decision to make but I wonder about the use of the word Refugee here...it potentially excludes asylum seekers/ anyone who has not been officially granted refugee status for whatever reason?

**Review by eliazzo — COMMENTED** [review]
Typo on line 73 .github/CONTRIBUTING.md
If you have a question or suggetion* directly related to an open Issue please use the comments on that Issue ticket to discuss it with anyone working on it / the core team.

*suggestion

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (camelPhonso) [reply .github/CONTRIBUTING.md:28]: 🏴 Uh, that's actually a good flag.

Throughout Kindly we use the word 'refugee' (for example right on the welcome page before signup) so I just kind of default to it in the assumption that came from the PO originally.

We should really just make sure that we're aligning everywhere with the language Trafalgar Girls uses to communicate with their community though - @Duncankatia is this something we should be adjusting on the platform and documentation?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+## User Stories
+The following User Stories follow the projected Roadmap shown above.
+When working on any Issue collaborators are encouraged to look for additional improvement opportunities that will fullfil these stories.
+Similarly, new Issue submissions are always welcomed but may be prioritized for fulfillment if they conform to any of the stories below.
+
+### User Account Milestone 🗝️
+- As a User I want to have an option to reset my password from the login page
+- As a User I want to be able to suspend my account
+- As a User I want to be able to delete my account
+- As a User I want to be able to browse through Kindly before I need to create an account in order to interact Donations
```
[the remark this answers is not in the document]
Comment (camelPhonso) [reply .github/CONTRIBUTING.md]: Fixed in a new commit 👍

File: README.md (hunk 1 above)
Comment (camelPhonso) [reply README.md]: Fixed in a new commit 👍

**Review by eliazzo — APPROVED** [review]

**Pull request merged**